src/runner/index.js

```diff
@@ -156,7 +155,13 @@ export default class Runner extends EventEmitter {
     }
 
     src (...sources) {
-        sources = flatten(sources).map(path => resolvePath(path));
+        // NOTE: the behavior here is consistent with the CLI
+        // src([]) won't search for tests in default folders, while
+        // src() will.
+        sources = arguments.length === 0 ? void 0 : flatten(sources);
+
+        if (!sources || sources.length)
+            sources = parseFileList(sources, process.cwd());
```

Comment:
We can't add default test folders here. Take a look at the line below:
```js
    this.bootstrapper.sources = this.bootstrapper.sources.concat(sources);
```

It means you can chain `src()` calls:
```js
    .src()
    .src()
    .src()
```
If we handle default folders in the `src` method, they will be added 3 times to the sources. 

So we should move the logic for handling default folders to the Bootstrapper class.